docs/chat/chat.yaml

```diff
@@ -0,0 +1,42 @@
+paths:
+  /chats:
+    post:
+      security:
+        - bearerAuth: []
+      tags:
+        - Messages
```

Comment:
shouldn't it be Chats tags instead of Messages

Reply:
Fixed.